src/translations/strings.yaml

```diff
@@ -568,7 +568,7 @@ onboarding:
     value: "Onboarding step “%1” (%2 of %3). You are currently on this step."
     comment: Accessibility read out for the current step in the step progress bar. %1 is the name of the step. %2 is the index of the current step. %3 is the total number of steps.
   progressBarAccessibilityStepIncomplete:
-    value: "Onboarding step “%1” (%2 of %3)."
```

Comment:
Do we need this change, given it's an accessibility label? This will force retranslation in all locales, and it doesn't seem useful.

Reply:
This is the fix for issue 2 in the PR description. The Windows screen reader reads the period as "Dot" because it is confused by the preceding ")". So, I feel that we need this change because it affects the Acessibility experience.